### PR TITLE
resolveImage helper. using params and not / for paths so windows works

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

**Comment by cjb** [comment]
:thumbsup:

**Comment by chrisnojima** [comment]
actually have more fixes, stay tuned

**Comment by chrisnojima** [comment]
@keybase/react-hackers ready for feedback again

File: desktop/resolve-root.js [hunk 1]
```diff
@@ -12,4 +12,13 @@ if (!__DEV__) { // eslint-disable-line no-undef
   root = path.join(__dirname)
 }
 
-export default (...to) => path.resolve(root, ...to)
+function fixSep (str) {
+  return str && str.replace(new RegExp('\\' + path.sep, 'g'), '/')
+}
+
+export const resolveRoot = (...to) => path.resolve(root, ...to)
```
Comment (chrisnojima) [inline desktop/resolve-root.js]: we now understand about file paths and urls so we can swap the \ / around

Comment (chrisnojima) [inline desktop/resolve-root.js]: and since images are a big use case we have a helper for that

**Comment by cjb** [comment]
:thumbsup:

**Pull request merged**